src/strategies/peak_load_window.py

```diff
@@ -83,6 +84,10 @@ def step(self, event_list=[]):
         standing = {}
         energy_needed = 0
         for vid, v in self.world_state.vehicles.items():
+            if v.vehicle_type.v2g:
+                warnings.warn("The vehicle type of vehicle {} is set to v2g although v2g is not "
+                              "supported with peak_load_window strategy. V2g will thus be "
+                              "neglected.".format(vid))
```

Comment:
I would prefer the capitalization of V2G (instead of lower-case letters), as it is an abbreviation.